db/sequence_hasher.go

```diff
@@ -258,14 +258,14 @@ func (s *sequenceHasher) GetClock(sequence string) (*base.SequenceClockImpl, err
 		return clock, loadErr
 	}
 
+	clock = base.NewSequenceClockImpl()
 	if uint16(len(storedClocks.Sequences)) <= seqHash.collisionIndex {
-		base.LogTo("Changes+", "Stored hash not found for sequence [%s] collision index [%d], #storedClocks:%d", sequence, seqHash.collisionIndex, len(storedClocks.Sequences))
-		return clock, errors.New(fmt.Sprintf("Stored hash not found for sequence [%s], returning zero clock", sequence))
+		base.LogTo("Changes+", "Stored hash not found for sequence [%s] collision index [%d], #storedClocks:%d, returning zero clock", sequence, seqHash.collisionIndex, len(storedClocks.Sequences))
```

Comment:
If we're not returning any information to the caller, this should be logging a warning (and not just a log key that might not be enabled).